setup.py

```diff
@@ -28,7 +28,6 @@
         'python-dateutil'
     ],
     dependency_links=[
-        'https://github.com/lystable/PyGithub/archive/ca6d43eb3b6ee14637940988fd4ac7eb3c207c79.zip#egg=PyGithub',
```

Comment:
the dependency is already specified in the requirements above. It doesn't need an explicit link if it's on PyPI